scripts/test-extensions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
```

Comment:
Please add tracking task on enabling this in our Azure DevOps validation builds (and code coverage).

Reply:
https://github.com/Microsoft/azuredatastudio/issues/4099